composer.json

```diff
@@ -7,14 +7,39 @@
 		"bamarni/composer-bin-plugin": "1.8.2",
 		"dealerdirect/phpcodesniffer-composer-installer": "0.7.2",
 		"dms/phpunit-arraysubset-asserts": "0.4.0",
+		"humbug/php-scoper": "0.13.9",
+		"pelago/emogrifier": "6.0.0",
 		"phpcompatibility/phpcompatibility-wp": "2.1.4",
 		"sirbrillig/phpcs-no-get-current-user": "1.1.0",
 		"sirbrillig/phpcs-variable-analysis": "2.11.9",
 		"squizlabs/php_codesniffer": "3.7.1",
 		"wp-coding-standards/wpcs": "2.3.0",
 		"yoast/phpunit-polyfills": "1.0.4"
 	},
+	"minimum-stability": "dev",
```

Comment:
Lowering the minimum stability because `php-scoper` has a `dev-master` dependency.